### PR TITLE
Sagar/dbproxy

### PR DESCRIPTION
**Pull request by cxvisa**

### CODE REVIEW

File: helm/db-controller/templates/certs.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{{ if .Values.dbproxy.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: dbproxy-selfsigned-issuer
```
Comment (daniel-garcia) [inline helm/db-controller/templates/certs.yaml:5]: namespaced

File: helm/db-controller/templates/certs.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  namespace: {{ .Release.namespace }}
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: db-controller-webhook-cert
```
Comment (daniel-garcia) [inline helm/db-controller/templates/certs.yaml:13]: namespaced

File: helm/db-controller/templates/webhook.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{{ if .Values.dbproxy.enabled }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: dbproxy-injector-webhook
```
Comment (daniel-garcia) [inline helm/db-controller/templates/webhook.yaml:5]: namespaced

File: helm/db-controller/templates/certs.yaml (hunk 1 above)
Comment (cxvisa) [reply helm/db-controller/templates/certs.yaml]: Using db-controller.fullname

File: helm/db-controller/templates/certs.yaml (hunk 2 above)
Comment (cxvisa) [reply helm/db-controller/templates/certs.yaml]: Using db-controller.fullname

File: helm/db-controller/templates/webhook.yaml (hunk 3 above)
Comment (cxvisa) [reply helm/db-controller/templates/webhook.yaml]: Using db-controller.fullname

**Review by bjeevan-ib — APPROVED** [review]

**Pull request merged**